pkg/apis/application/v1alpha1/app_catalog_types.go

```diff
@@ -46,21 +46,35 @@ func NewAppCatalogCRD() *apiextensionsv1beta1.CustomResourceDefinition {
 // +genclient
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
+// AppCatalog CR example as below.
+//
+//     apiVersion: application.giantswarm.io/v1alpha1
+//     kind: AppCatalog
+//     metadata:
+//       name: "giant-swarm"
+//     spec:
+//       title: "Giant Swarm"
+//       description: “Catalog of Apps by Giant Swarm."
+//       catalogStorage:
+//         type: "helm"
+//         URL: "https://github.com/giantswarm/app-catalog/"
```

Comment:
Apparently previous version was correct. It is a github page.

```suggestion
//         URL: "https://giantswarm.github.com/app-catalog/"
```